tripper/triplestore.py

```diff
@@ -495,6 +495,32 @@ def prefix_iri(self, iri: str, require_prefixed: bool = False):
                 raise NamespaceError(f"No prefix defined for IRI: {iri}")
         return iri
 
+    def map(
```

Comment:
Why is there a method that just calls another method instead of just calling the other method directly?